tests/angular-js/e2e/alternativeHelp.js

```diff
@@ -14,7 +14,7 @@
   var modalHeading = modal.element(by.css('h2'));
   var modalSubmit = modal.element(by.css('button[type="submit"]'));
 
-  describe('alternativeHelp', function () {
+  ddescribe('alternativeHelp', function () {
```

Comment:
The JSHint gulp task should pick these up actually. We were tempted to have tests fail if JSHint failed at one point.